packages/config/src/forkConfig/index.ts

```diff
@@ -85,22 +85,49 @@ export function createForkConfig(config: ChainConfig): ForkConfig {
   const forksAscendingEpochOrder = Object.values(forks);
   const forksDescendingEpochOrder = Object.values(forks).reverse();
 
+  const blobScheduleDescendingEpochOrder = [...config.BLOB_SCHEDULE].sort((a, b) => b.EPOCH - a.EPOCH);
+
+  const forkBoundariesAscendingEpochOrder: ForkBoundary[] = [
```

Comment:
In the case where a bpo fork (B) and hard fork (H) have the same epoch (eg. first BPO fork is scheduled at the same epoch as fulu's activation epoch), we need a tie breaker when sorting. The `.sort` here does not guarantee order whether it be B then H or H then B

The logic in `getActiveForkBoundaries()`, `createCachedGenesis()` and `getCurrentAndNextForkBoundary()` expects `forkBoundariesAscendingEpochOrder` to be H then B, so we need to enforce this ordering here.